app/views/admin/maintenance_tasks/index.html.haml

```diff
@@ -5,19 +5,21 @@
 
 %h1= t '.title'
 
-%table.table.table-striped
-  %tr
-    %th{style: "width: 150px;"}= t '.maintenance_task'
-    %th= t :description
-    %th{colspan: "3"}= t :actions
-  - @maintenance_tasks.each do |task|
-    %tr{class: "#{cycle('odd', 'even')}"}
-      %td= task.label
-      %td= task.description
-      - if task.prepare?
-        %td= link_to t('admin.maintenance_tasks.index.prepare'), admin_maintenance_tasks_prepare_path(task.id)
-      - else
+- if !@maintenance_tasks.empty?
```

Comment:
Negation is not the ruby way. Maybe use [any?](https://apidock.com/ruby/Enumerable/any%3F) or [unless](https://www.tutorialspoint.com/ruby/ruby_if_else.htm)